beacon-chain/casper/incentives.go

```diff
@@ -13,14 +13,20 @@ var log = logrus.WithField("prefix", "casper")
 // based on FFG incentive structure.
 func CalculateRewards(attestations []*pb.AggregatedAttestation, validators []*pb.ValidatorRecord, dynasty uint64, totalDeposit uint64) ([]*pb.ValidatorRecord, error) {
 	activeValidators := ActiveValidatorIndices(validators, dynasty)
-	attesterDeposits := GetAttestersTotalDeposit(attestations)
-
+	attesterDeposits := AttestersTotalDeposit(activeValidators)
+	attesterBitfield := attestations[len(attestations)-1].AttesterBitfield
 	attesterFactor := attesterDeposits * 3
 	totalFactor := uint64(totalDeposit * 2)
+
+	// Pad 0s to the rest of attesterBitfield bitfield up to default validator length.
+	for len(attesterBitfield)*8 < params.BootstrappedValidatorsCount {
```

Comment:
Shouldn't `attesterBitfield` already be padded with 0s because it was validated? Specifically [`AreAttesterBitfieldValid`](https://github.com/prysmaticlabs/prysm/blob/master/beacon-chain/casper/validator.go#L115).

Reply:
Just verified that we don't need this. I'll take it out